### DISCUSSION TITLE
Broken Windows services after tweaks

### DISCUSSION THREAD

**Discussion by jedenzet** [ChrisTitusTech/winutil#761] [Q&A]
Hello.
Latest version of this utility literally breaks windows services. I'm unable to start Security Center service and i can't even use a "taskkill" command. I'm also having problems with installing programs for example "elgato wavelink" that "terminates prematurely because of error". 
Can anyone help me? I'm on a fresh windows install and i don't want to do it again.

**Comment by gmy77** [comment]
About me... the function to set all SERVICES to MANUAL... is very... problematic at least. I stay away from this selection, but... hey this is only my view.
